app/views/registration_wizard/have_ofsted_urn.html.erb

```diff
@@ -1,44 +1,16 @@
-<% content_for :title do %>
-  <%= @form.errors.present? ? "Error: " : nil %>Do you or your employer have an Ofsted unique reference number (URN)?
+<% content_for :additional_info do %>
```

Comment:
I am happy to rename `:additional_info` for more meaningful name.

Reply:
@BroiSatse I recall you implementing something similar to this?

Reply:
Yeah I did. There are two similiar concepts in already - one is `body` attribute (which is unable to handle this case) and the other one is a slot on Question as a ViewComponent. I think I'm ok with this bein done as is but I'd like to o over all the steps at one point and clean it all up.